test/e2e/helloworld_test.go

```diff
@@ -441,6 +441,7 @@ func prepareAddOnDeploymentConfig(namespace string) error {
 						NodeSelector: nodeSelector,
 						Tolerations:  tolerations,
 					},
+					AgentInstallNamespace: metav1.NamespaceDefault,
```

Comment:
test/e2e/helloworld_helm_test.go also calls `prepareAddOnDeploymentConfig` and `prepareImageOverrideAddOnDeploymentConfig`, need to update the corresponding namespace when checking the generated resource.